CompanyEmployees.Application/Commands/CheckInAttendanceCommand.cs

```diff
@@ -0,0 +1,6 @@
+using MediatR;
+using Shared.DataTransferObjects;
+
+namespace CompanyEmployees.Application.Commands;
+
```

Comment:
The semicolon after the namespace declaration is unnecessary and non-idiomatic in C#. Remove it to follow standard namespace syntax.

```suggestion
namespace CompanyEmployees.Application.Commands
```